src/servers/main.cc

```diff
@@ -265,18 +266,32 @@ std::vector<Option> options_{
     {OPTION_TRACE_RATE, "trace-rate",
      "Set the trace sampling rate. Default is 1000."},
 #endif  // TRTIS_ENABLE_TRACING
+    {OPTION_MODEL_CONTROL_MODE, "model-control-mode",
+     "Specify the mode for model management. Options are \"none\", \"poll\" "
+     "and \"explicit\". "
+     "For \"none\", the server will load all models in the model repositories "
+     "only once at startup. For \"poll\", the server will poll the model "
+     "repository to detect changes. The poll rate is controlled by "
+     "'repository-poll-secs'. For \"explicit\", the model load and unload is "
+     "initiated by using the model control APIs, and the models in the model "
+     "repository will not be loaded at startup, unless the model is specified "
+     "by --load-model."},
     {OPTION_ALLOW_POLL_REPO, "allow-poll-model-repository",
-     "Poll the model repository to detect changes. The poll rate is "
-     "controlled by 'repository-poll-secs'."},
+     "[DEPRECATED] Poll the model repository to detect changes. The poll rate "
+     "is controlled by 'repository-poll-secs'. This option is deprecated by "
+     "--model-control-mode, this option will be overridden if "
```

Comment:
Here and elsewhere make the --allow options cause a failure if they are specified when --model-control-mode is specified.